api/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -134,13 +134,50 @@ func (in *DatabaseStatus) DeepCopy() *DatabaseStatus {
 // DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
 func (in *Instance) DeepCopyInto(out *Instance) {
 	*out = *in
+	if in.DatabaseInstanceName != nil {
+		in, out := &in.DatabaseInstanceName, &out.DatabaseInstanceName
+		*out = new(string)
+		**out = **in
+	}
 	if in.DatabaseNames != nil {
 		in, out := &in.DatabaseNames, &out.DatabaseNames
-		*out = make([]string, len(*in))
-		copy(*out, *in)
+		*out = new([]string)
+		if **in != nil {
```

Comment:
these 4-5 lines of code are not very intuitive. possible to use diff variable names? or explain what are we doing here?